### PR TITLE
Add upcoming event message to event detail page

### PR DESCRIPTION
**Pull request by aralatpulat**
#35

### CODE REVIEW

File: Resources/Views/event.leaf [hunk 1]
```diff
@@ -35,7 +35,11 @@
     <div class="row">
         <div class="col-sm">
             <div class="no-content-container" style="text-align: center">
+                #if(isUpcoming){
+                <b>#(event.fullname)</b> will take place on <b>#date(event.startDate, "yyyy/MM/dd")</b>. There are no videos for the event right now.
```
Comment (ahmetws) [inline Resources/Views/event.leaf]: indentaton

File: Resources/Views/event.leaf [hunk 2]
```diff
@@ -35,7 +35,11 @@
...
+                } else {
                 There are no videos for <b>#(event.fullname)</b> right now.<br/> We are working hard to add new videos and we hope they will be here soon.
```
Comment (ahmetws) [inline Resources/Views/event.leaf]: indentaton

File: Resources/Views/event.leaf [hunk 3]
```diff
@@ -35,7 +35,11 @@
     <div class="row">
         <div class="col-sm">
             <div class="no-content-container" style="text-align: center">
+                #if(isUpcoming){
```
Comment (ahmetws) [inline Resources/Views/event.leaf]: space after )

File: Resources/Views/videosSection.leaf [hunk 4]
```diff
@@ -3,7 +3,11 @@
         <div class="row">
             <div class="col-sm">
                 <div class="section-header-container">
+                    #if(isUpcoming){
```
Comment (ahmetws) [inline Resources/Views/videosSection.leaf]: space after (

File: Resources/Views/videosSection.leaf [hunk 5]
```diff
@@ -3,7 +3,11 @@
...
+                    <h2>Upcoming Videos<h2>
+                    } else{
```
Comment (ahmetws) [inline Resources/Views/videosSection.leaf]: space after else

File: Sources/App/routes.swift [hunk 6]
```diff
@@ -154,7 +154,7 @@ public func routes(_ router: Router) throws {
                 }
 
                 return apiClient.getEventVideos(db, eventId: eventId).flatMap({ (videos) -> EventLoopFuture<View> in
-                    let context = EventContext(videos: videos, event: event!)
+                    let context = EventContext(videos: videos, event: event!, isUpcoming: event!.isUpcoming)
```
Comment (ahmetws) [inline Sources/App/routes.swift]: should we use ! for the event

**Review by ahmetws — CHANGES_REQUESTED** [review]
Merge request title should explain the work done in the merge.

File: Sources/App/routes.swift [hunk 7]
```diff
@@ -149,12 +149,16 @@ public func routes(_ router: Router) throws {
 
         return apiClient.getEvent(db, shortUrl: value)
             .flatMap({ (event) -> EventLoopFuture<View> in
-                guard let eventId = event?._id else {
+                guard let event = event else {
+                    return try req.view().render("404")
+                }
+                
+                guard let eventId = event._id else {
```
Comment (ahmetws) [inline Sources/App/routes.swift]: please combine this two in a guard

Comment (aralatpulat) [reply Sources/App/routes.swift]: combined.

**Review by ahmetws — APPROVED** [review]

**Pull request merged**